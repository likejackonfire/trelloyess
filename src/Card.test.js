import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

describe('Card components', () => {
it("renders without crashing", () => {
    // Create a DOM element to render the component into
    const div = document.createElement("div");
  
    // Render the component
    // If something is wrong it will fail here
    ReactDOM.render(<Card />, div);
    // Clean up
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders the UI as expected", () => {
      const tree = renderer
      .create(<Card name="Card" />)
      .toJSON();
      expect(tree).toMatchSnapshot();
  });

});