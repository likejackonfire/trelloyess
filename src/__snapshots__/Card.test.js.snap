// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card components renders the UI as expected 1`] = `
<div
  className="Card"
>
  <p />
  <button
    onClick={[Function]}
    type="button"
  >
    Delete
  </button>
  <h3 />
  <p />
</div>
`;

exports[`renders the UI as expected 1`] = `
<div
  className="Card"
>
  <p />
  <button
    onClick={[Function]}
    type="button"
  >
    Delete
  </button>
  <h3 />
  <p />
</div>
`;
